contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -329,4 +394,4 @@ def _validate(
 
 
 if __name__ == "__main__":
-    ExpectColumnValuesToBePresentInAnotherTable().print_diagnostic_checklist()
+    ExpectColumnValuesToBePresentInOtherTable().print_diagnostic_checklist()
```

Comment:
Just a note that with V1, you can't instantiate without args like we can in 0.18 - this will fail on init

Reply:
Yup. This isn't included in the V1 version of this Expectation